modules/ROOT/pages/external-identity-index.adoc

```diff
@@ -3,11 +3,10 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-**** xref:managing-users.adoc[To Configure SAML]
-***** xref:verify-saml-info-task.adoc[To Verify SAML Information]
-***** xref:troubleshoot-saml-assertions-task.adoc[To Troubleshoot SAML Assertions]
-***** xref:update-user-profile-task.adoc[To Update Your User Profile]
-***** xref:single-log-out-task.adoc[To Configure Single Log Out]
-
-**** xref:conf-openid-connect-task.adoc[To Configure OpenID Connect]
-**** xref:test-external-identity-task.adoc[To Test External Identity Configuration]
\ No newline at end of file
+* xref:managing-users.adoc[Configure SAML for SSO]
+** xref:verify-saml-info-task.adoc[Verify SAML Information]
+** xref:single-log-out-task.adoc[Configure Single Log Out]
+** xref:troubleshoot-saml-assertions-task.adoc[Troubleshoot SAML Assertions]
+* xref:conf-openid-connect-task.adoc[Configure OpenID Connect]
+** xref:update-user-profile-task.adoc[Update Your User Profile]
+** xref:test-external-identity-task.adoc[Test External Identity Configuration]
```

Comment:
This should only have 1 asterisk so it is the same level as Configure SAML and Configure OpenId Connect.